doc/api.md

```diff
@@ -179,7 +179,7 @@ AdobeState get return object
 
 | Name | Type | Description |
 | --- | --- | --- |
-| expiration | <code>string</code> \| <code>null</code> | ISO date string of expiration time for the key-value pair, if the ttl is infinite expiration=null |
+| expiration | <code>number</code> \| <code>null</code> | time in milliseconds since epoch, for the key-value pair expiration=null |
```

Comment:
hmm, I think we should still expose it like in v2 for now?

Reply:
ok, I'll revert back to ISO string.

Reply:
fixed in https://github.com/adobe/aio-lib-state/pull/138/commits/d5fc278a03bfeae79dd1d23eaeeb3ba8917f4e14